### PR TITLE
Main menu code cleanup

### PR DESCRIPTION
**Pull request by Brycey92**

### CODE REVIEW

File: main/modes/mode_main_menu.c [hunk 1]
```diff
@@ -35,6 +35,19 @@
...
+// Defines
+//==============================================================================
+
+#define GIT_SHA1_SHORT_LENGTH 7
+
+// In case we want to use the full git SHA1 somewhere (change to 0 in CMakeLists.txt)
+#if GIT_SHA1_IS_SHORT
+#define GIT_SHA1_LENGTH 7
+#else
+#define GIT_SHA1_LENGTH 40
```
Comment (AEFeinstein) [inline main/modes/mode_main_menu.c:48]: `GIT_SHA1_LENGTH` isn't used anywhere so I don't know what's the point.

Comment (AEFeinstein) [inline main/modes/mode_main_menu.c:48]: `GIT_SHA1_LENGTH` isn't used anywhere so I don't know what's the point.

File: emu.mk [hunk 2]
```diff
@@ -115,6 +115,7 @@ CFLAGS_WARNINGS_EXTRA = \
 
 # Create a variable with the git hash and branch name
 GIT_HASH  = \"$(shell git rev-parse --short HEAD)\"
+GIT_HASH_IS_SHORT = 1
```
Comment (AEFeinstein) [inline emu.mk:118]: Don't add unrelated variables. If you're gonna add this, make it do something, like change the length of `GIT_HASH` or something.

**Review by AEFeinstein — CHANGES_REQUESTED** [review]

File: emu.mk (hunk 2 above)
Comment (Brycey92) [reply emu.mk:118]: That was originally the plan, but I realized it wouldn't fit where we have it on the easter egg submenu

Comment (AEFeinstein) [reply emu.mk:118]: A PR with "cleanup" in the title usually doesn't add unused code... Clean that up!

Comment (Brycey92) [reply emu.mk:118]: You are, as usual, entirely correct lol
Cleaning up now

**Review by AEFeinstein — APPROVED** [review]

**Pull request merged**